hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -261,6 +261,8 @@ MDMParser::MDMParser(void)
     _power_mode = 1; // default power mode is AT+UPSV=1
     _cancel_all_operations = false;
     sms_cb = NULL;
+    _memoryIssuePresent = true;
+    _oldFirmwarePresent = true;
```

Comment:
Let's remove the initialization of these member variables from the header file where they're declared – for consistency.

Reply:
Absolutely...